connector/throughputanomalyconnector/config.go

```diff
@@ -0,0 +1,78 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package throughputanomalyconnector
+
+import (
+	"fmt"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+)
+
+var _ component.Config = (*Config)(nil)
+
+// Config defines the configuration parameters for the log anomaly detector connector.
+type Config struct {
+	// How often to take measurements
+	SampleInterval time.Duration `mapstructure:"sample_interval"`
```

Comment:
We aren't really sampling anymore right? But this is used to control how often we perform analysis on the counts, so maybe there's a better name.

Related, but probably for a future enhancement - should this be a "WindowSize" setting so that we're analyzing each window once?